pkg/descheduler/strategies/pod_lifetime.go

```diff
@@ -106,15 +106,16 @@ func PodLifeTime(ctx context.Context, client clientset.Interface, strategy api.D
 	// in the event that PDB or settings such maxNoOfPodsToEvictPer* prevent too much eviction
 	podutil.SortPodsBasedOnAge(podsToEvict)
 
+	nodeLimitExceeded := map[string]bool{}
 	for _, pod := range podsToEvict {
-		success, err := podEvictor.EvictPod(ctx, pod, nodeMap[pod.Spec.NodeName], "PodLifeTime")
-		if success {
+		if nodeLimitExceeded[pod.Spec.NodeName] {
+			continue
+		}
+		if podEvictor.EvictPod(ctx, pod) {
 			klog.V(1).InfoS("Evicted pod because it exceeded its lifetime", "pod", klog.KObj(pod), "maxPodLifeTime", *strategy.Params.PodLifeTime.MaxPodLifeTimeSeconds)
 		}
-
-		if err != nil {
-			klog.ErrorS(err, "Error evicting pod", "pod", klog.KObj(pod))
-			break
+		if podEvictor.NodeLimitExceeded(nodeMap[pod.Spec.NodeName]) {
+			nodeLimitExceeded[pod.Spec.NodeName] = true
```

Comment:
could we add some logs for NodeLimitExceeded is true for easier to debug?

Reply:
```bash
$ grep -rn "NodeLimitExceeded"
pkg/framework/plugins/removepodsviolatingnodetaints/node_taint.go:118:				if d.handle.Evictor().NodeLimitExceeded(node) {
pkg/descheduler/strategies/node_affinity.go:97:						if podEvictor.NodeLimitExceeded(node) {
pkg/descheduler/strategies/failedpods.go:79:			if podEvictor.NodeLimitExceeded(node) {
pkg/descheduler/strategies/node_taint.go:111:				if podEvictor.NodeLimitExceeded(node) {
pkg/descheduler/strategies/pod_lifetime.go:137:		if podEvictor.NodeLimitExceeded(nodeMap[pod.Spec.NodeName]) {
pkg/descheduler/strategies/duplicates.go:204:					if podEvictor.NodeLimitExceeded(nodeMap[nodeName]) {
pkg/descheduler/strategies/toomanyrestarts.go:94:			if podEvictor.NodeLimitExceeded(node) {
pkg/descheduler/strategies/pod_antiaffinity.go:99:			if podEvictor.NodeLimitExceeded(node) {
pkg/descheduler/strategies/topologyspreadconstraint.go:195:		if podEvictor.NodeLimitExceeded(nodeMap[pod.Spec.NodeName]) {
pkg/descheduler/strategies/nodeutilization/nodeutilization.go:348:			if podEvictor.NodeLimitExceeded(nodeInfo.node) {
```

Would be great to locate other places worth extending with new logs. @JaneLiuL would you be interested in doing so? With the new framework and migration to plugins we will need to log more to improve the user experience.